src/main/webapp/app/exercises/shared/plagiarism/plagiarism-split-view/plagiarism-split-view.component.html

```diff
@@ -1,25 +1,17 @@
 <div class="plagiarism-split-view">
     <div class="plagiarism-split-pane" jhiPane>
-        <div class="plagiarism-split-pane-header">
-            {{ comparison.submissionA.studentLogin || ('artemisApp.plagiarism.unknown-student' | translate) }}
-        </div>
+        <jhi-modeling-submission-viewer *ngIf="isModelingExercise()" [exercise]="exercise" [plagiarismSubmission]="getModelingSubmissionA()"></jhi-modeling-submission-viewer>
```

Comment:
It is better if you don't call methods directly from the HTML because it will continuously trigger the function. You can achieve better performance if you store them as attributes in the component.

Reply:
I updated the calls to isModelingExercise() etc. to be attributes of the components instead. 
But I still call getModelingSubmissionA() etc. in the template as this method is just a typecast and I don't know how to do it in the template directly.